openmdao/solvers/solver.py

```diff
@@ -365,12 +372,21 @@ def _iter_initialize(self):
             error at the first iteration.
         """
         if self.options['maxiter'] > 0:
+            self._run_apply()
             norm = self._iter_get_norm()
         else:
             norm = 1.0
         norm0 = norm if norm != 0.0 else 1.0
         return norm0, norm
 
+    def _run_apply(self):
+        """
+        Run the the apply_nonlinear method on the system.
+        """
+        recording_iteration_stack.append(('_run_apply', 0))
```

Comment:
This used to be `get_iter_norm`, and I renamed it to `run_apply` because I moved that code. This tag on the stack is used to prevent recording during this phase, so If I removed this, it would presumably print some extra cases, though I can only say this based on design intent, and not on viewing any case output.